lib/game.rb

```diff
@@ -92,7 +92,7 @@ def is_a_tie?
     @board.full? && !player_wins?(@current_player.mark)
   end
 
-  def outcome
+  def show_outcome
     system('clear')
```

Comment:
`system('clear')` seems to operate at a lower level of abstraction than the surrounding code. How could you make sure this clearing the screen operates at the same level of abstraction as `@display_show_board`?

If you catch my drift ... :))